app/uk/gov/hmrc/eeittadminfrontend/controllers/DeploymentController.scala

```diff
@@ -344,8 +343,7 @@ class DeploymentController @Inject() (
       }
     } match {
       case Some(ids) if ids.nonEmpty => Right(ids.toList)
-      case Some(_)                   => Left("No valid handlebar schema IDs found.")
-      case None                      => Left("Unable to find any destinations for the json schema.")
+      case _                         => Right(Nil)
```

Comment:
`Left` is no longer used, we should change return type of `getHandlebarsSchemaIds ` to be `List[String]` only

Reply:
also `if ids.nonEmpty` guard seems to be redundant now.